SecureTrading/SecureTradingUI/Source Files/Common/Views/Form/CardNumberInputView.swift

```diff
@@ -27,6 +38,14 @@ import UIKit
         return CardNumber(rawValue: textFieldTextWithoutSeparators)
     }
 
+    @objc public var cardType: CardType {
+        return CardValidator.cardType(for: cardNumber.rawValue, cardTypes: cardTypeContainer.cardTypes)
+    }
+
+    @objc public var isRequiredCvc: Bool {
+        return CardValidator.isCVCRequired(for: cardType)
+    }
+
     @objc public override var inputIsValid: Bool {
```

Comment:
I think isInputValid would sound better, now it's not clear if it states or asks